src/test/java/eu/dissco/backend/TestUtils.java

```diff
@@ -34,7 +31,10 @@ public class TestUtils {
   public static final String ID_ALT = PREFIX + "/" + "AAA-111-ZZZ";
   public static final String TARGET_ID = PREFIX + "/TAR_GET_001";
 
-  public static final ObjectMapper MAPPER = new ObjectMapper().findAndRegisterModules();
+  public static final ObjectMapper MAPPER = new ObjectMapper()
+      .findAndRegisterModules()
+      .setDateFormat(new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ"))
```

Comment:
Did you get this working? It still gave to many SSSSS's for me

Reply:
All it took was 3 different de/serializers :)

Reply:
technically 2.5

Reply:
Nice, also looked at a couple of blogs but it seems to also add the nanoseconds in the Jackson Instant Serializer (https://www.javadoc.io/static/com.fasterxml.jackson.datatype/jackson-datatype-jsr310/2.6.5/index.html?com/fasterxml/jackson/datatype/jsr310/ser/InstantSerializer.html), strange... But this seems to work nicely :+1: